src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,183 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+Commands to manage Azure container registries
+-------------
+::
+
+    Group
+        az acr: Commands to manage Azure container registries.
+
+    Subgroups:
+        credential: Manage admin user credential for Azure container registries.
+        repository: Manage repositories for Azure container registries.
+        storage   : Manage storage accounts for Azure container registries.
+
+    Commands:
+        create    : Create a container registry.
+        delete    : Delete a container registry.
+        list      : List container registries.
+        show      : Get a container registry.
+        update    : Update a container registry.
+
+Create a container registry
+-------------
+::
+
+    Command
+        az acr create: Create a container registry.
+
+    Arguments
+        --location -l       [Required]: Location.
```

Comment:
We are only supporting one location currently (but will increase) and it is very unlikely that the location for the resource group would be the one that we support. I would keep it as required for now.